contrib/opencensus-ext-azure/opencensus/ext/azure/common/storage.py

```diff
@@ -170,3 +170,25 @@ def put(self, data, lease_period=0, silent=False):
             ),
         ))
         return blob.put(data, lease_period=lease_period, silent=silent)
+
+
+class LocalNoopStorage(object):
```

Comment:
This is like a mock class for testing, do we want to hack this way?

Reply:
Refactored to exporter to include _transmit functionality without retry policy.